### PR TITLE
Added spaces to imports

### PR DESCRIPTION
**Pull request by NejcZdovc**
Related to the issue #69.

### CODE REVIEW

File: src/config-serializer/config-items/key-action/NoneAction.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import {UhkBuffer} from '../../UhkBuffer';
-import {KeyAction, KeyActionId, keyActionType} from './KeyAction';
+import { UhkBuffer} from '../../UhkBuffer';
```
Comment (fjozsef) [inline src/config-serializer/config-items/key-action/NoneAction.ts]: A space is missing here

**Comment by NejcZdovc** [comment]
@fjozsef please review

File: src/config-serializer/config-items/key-action/NoneAction.ts (hunk 1 above)
Comment (NejcZdovc) [inline src/config-serializer/config-items/key-action/NoneAction.ts]: Done

File: src/components/popover/tab/mouse/mouse-tab.component.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
-import {Component, Input, OnInit} from '@angular/core';
+import { Component, Input, OnInit } from '@angular/core';
 
-import {KeyAction, MouseAction, MouseActionParam} from '../../../../config-serializer/config-items/key-action';
-import {Tab} from '../tab';
+import { KeyAction, MouseAction, MouseActionParam } from '../../../../config-serializer/config-items/key-action';
+import { Tab} from '../tab';
```
Comment (fjozsef) [inline src/components/popover/tab/mouse/mouse-tab.component.ts]: A missing space here, too

Comment (NejcZdovc) [inline src/components/popover/tab/mouse/mouse-tab.component.ts]: Done

**Pull request merged**